README.md

```diff
@@ -5,3 +5,9 @@ Currently in development and not yet ready to be used.
 ## Installing from source
 `pip3 install .`
 
+## Running tests
+1. Install `pytest` from `pip`: \
+```pip3 install pytest```
+2. Install `sinol-make`: \
+```pip3 install .```
```

Comment:
Can't we add `pytest` as a dependency, so it also gets downloaded during `pip3 install .`?

Reply:
Sure, I changed it: 60ec487e1ce8d893f3fd8545666eae90e9b4da93